test/e2e/storage/csi_volumes.go

```diff
@@ -61,7 +61,9 @@ var _ = utils.SIGDescribe("CSI Volumes", func() {
 
 	Context("CSI Topology test using GCE PD driver [Feature:CSINodeInfo]", func() {
 		f := framework.NewDefaultFramework("csitopology")
-		driver := drivers.InitGcePDCSIDriver(true /* topology enabled */).(testsuites.DynamicPVTestDriver) // TODO (#71289) eliminate by moving this test to common test suite.
+		//driver := drivers.InitGcePDCSIDriver(true /* topology enabled */).(testsuites.DynamicPVTestDriver) // TODO (#71289) eliminate by moving this test to common test suite.
```

Comment:
same thing I don't think we need to comment these out

Reply:
Okay